src/Umbraco.Web.UI.Client/src/views/documenttypes/create.html

```diff
@@ -81,18 +81,20 @@ <h5><localize key="create_createUnder">Create an item under</localize> {{current
 
                 <umb-control-group label="Name of the Parent Document Type" hide-label="false">
                     <input type="text" name="collectionName" ng-model="model.collectionName" class="umb-textstring textstring input-block-level" umb-auto-focus required />
-                    <span ng-if="model.disableTemplates === false">
-                    <input id="collectionCreateTemplate" name="collectionCreateTemplate" type="checkbox" ng-model="model.collectionCreateTemplate" style="margin-top: 0;" />
-                    <label for="collectionCreateTemplate" style="margin-bottom: 0; padding-left: 2px;">Create template for the Parent Document Type</label>
-                    </span>
+
+                    <div ng-if="model.disableTemplates === false" class="flex">
+                        <umb-checkbox input-id="collectionCreateTemplate" name="collectionCreateTemplate" model="model.collectionCreateTemplate" />
```

Comment:
Hey @bjarnef - The label is not neccessary when you use the umb-checkbox directive. The directive puts in a label for you. All that is really needed is this

```
                   <umb-checkbox
                        ng-if="model.disableTemplates === false"
                        model="model.collectionCreateTemplate"
                        text="Create template for the Parent Document Type"
                    />
```
In this case I have added the `ng-if` check on the directive as well - Currently the checkbox will render with two labels, which is not ideal for accessibility and the `input-id` and `name` attribues are not neccessary 😃